pkgs/by-name/ku/kubectl-rolesum/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "kubectl-rolesum";
+  version = "1.5.6";
+
+  src = fetchFromGitHub {
+    owner = "Ladicle";
+    repo = "kubectl-rolesum";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-Nj4LyAruyDyX9NR0bTQzuAaLPBZNLeVWilobGlHop/o=";
+  };
+
+  vendorHash = "sha256-nt/GOG4kdUrmOsMnFs6fNwiBZDVXoa7OdwnZsD2cPy8=";
+
+  ldflags = [
+    "-X github.com/Ladicle/kubectl-rolesum/cmd.version=${finalAttrs.version}"
+  ];
+
+  versionCheckProgramArg = [ "--version" ];
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Summarize Kubernetes RBAC roles for the specified subjects";
+    mainProgram = "kubectl-rolesum";
+    homepage = "https://github.com/Ladicle/kubectl-rolesum";
+    changelog = "https://github.com/Ladicle/kubectl-rolesum/releases/tag/v${finalAttrs.version}";
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.tboerger ];
```

https://github.com/Ladicle/kubectl-rolesum/blob/v1.5.6/.goreleaser.yml#L12-L15


I suggest adding `meta.platforms`.

For example:

```suggestion
    maintainers = [ lib.maintainers.tboerger ];
    platforms = with lib.platforms; unix ++ windows;
```